### PR TITLE
Issue 1234 boxfill anim broken

### PR DESCRIPTION
**Pull request by doutriaux1**
@aashish24 @dlonie @chaosphere2112 goes with:
https://github.com/UV-CDAT/uvcdat-testdata/pull/37

### CODE REVIEW

File: Packages/vcs/Lib/VTKPlots.py [hunk 1]
```diff
@@ -1153,7 +1153,7 @@ def plot2D(self,data1,data2,tmpl,gm,vtk_backend_grid=None,vtk_backend_geo=None):
       else:
         lmx= levs[-1]
       mapper.SetScalarRange(lmn,lmx)
-      returned["vtk_backend_luts"]=[[lut,[lmn,lmx,False]],]
+      returned["vtk_backend_luts"]=[[lut,[lmn,lmx,True]],]
```
Comment (dlonie) [inline Packages/vcs/Lib/VTKPlots.py]: These map entries really need some documentation. I have no idea what this change should do.

**Comment by dlonie** [comment]
Building/testing now.

**Comment by aashish24** [comment]
hey guys.. not picking on anyone but please use better short messages for the commit.

**Comment by aashish24** [comment]
@dlonie I am assuming that you will have some feedback on it.

**Comment by dlonie** [comment]
Yep, still building. Assuming all goes well this can go in -- we can work on documenting that map when the animation code gets hit by the refactoring sweep.

**Comment by aashish24** [comment]
:+1: in general we should document our code as much as possible atleast when something is not trivial or  error prone. In my document, I will make it a requirement.

**Comment by dlonie** [comment]
LGTM, tests pass.

File: Packages/vcs/Lib/VTKPlots.py (hunk 1 above)
Comment (doutriaux1) [inline Packages/vcs/Lib/VTKPlots.py]: yes but aren't we changing all of this anyway once we refactor?

**Pull request merged**